src/Interactive/EditorFeatures/Core/InteractiveEditorFeaturesResources.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ReferencesCommandDescription" xml:space="preserve">
```

Comment:
These are actually for a subsequent change, but I suppose there's no harm in merging them now.